igvc2023/config/navsatekf.yaml

```
@@ -1,10 +1,10 @@
 frequency: 30 # 30
 delay: 0
 magnetic_declination_radians: 0 # 0
-yaw_offset: 0.0
+yaw_offset: 1.5707963
 zero_altitude: false # false
-broadcast_utm_transform: ture # false
-publish_filtered_gps: ture
+broadcast_utm_transform: true # false
```

```
navsat_transform, Parameter 'broadcast_utm_transform' has been deprecated. Please use'broadcast_cartesian_transform' instead.
```
noeticだとクラッシュすることはないようですが、もし上記のようなwarningが出るようであれば
```suggestion
broadcast_cartesian_transform: true # false
```
のように変更した方が良いと思います。

参考までに: https://github.com/cra-ros-pkg/robot_localization/issues/732